flash_examples/finetuning/image_classification.py

```diff
@@ -27,16 +30,30 @@
     test_folder="data/hymenoptera_data/test/",
 )
 
-# 3. Build the model
-model = ImageClassifier(num_classes=datamodule.num_classes)
 
-# 4. Create the trainer.
+# 3. a Register a custom backbone
+@ImageClassifier.register_function(name="username/resnet18")
+def fn_resnet(pretrained: bool = True):
+    model = getattr(torchvision.models, "resnet18", None)(pretrained)
+    backbone = nn.Sequential(*list(model.children())[:-2])
```

Comment:
add a comment for what this line does for newcomers

Reply:
Done !